api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,56 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a Subscription for EventType(s). The subscription is needed to be able to consume
+        events from EventType(s) in a high level way when Nakadi stores the offsets and manages the rebalancing of
+        consuming clients.
+        The subscription is identified by its parameters. If this endpoint is invoked several times with the same
+        subscription properties in body - the subscription will be created only once and for all calls it will
+        return the same subscription id.
+      parameters:
+        - name: subscription
+          in: body
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: true
+      responses:
+        '200':
```

Comment:
My problem here is that we don't obviously create a subscription, in a case when one already exists we just return an id. So I'm not sure if we can always return 201.
Maybe we can return 201 if it is really created and 200 if we just return id?